src/features/book/components/BookLogList.tsx

```
@@ -0,0 +1,246 @@
+                <br />
+                독서하는 순간에 떠오르는 생각을 기록해보세요!
+              </p>
+            </div>
+          ) : (
+            <div className="flex flex-col gap-xlarge">
+              {allRecords.map((item, idx) => {
+                switch (item.type) {
+                  case 'personal':
+                    return (
+                      <PersonalRecordItem
+                        key={`personal-${item.data.recordId}`}
```

Biome (2.3.13)

[error] This callback passed to map() iterable method should always return a value.

Add missing return statements so that this callback returns a value on all execution paths.

(lint/suspicious/useIterableCallbackReturn)